packages/wrangler/src/completions/scripts.ts

```diff
@@ -0,0 +1,90 @@
+import type { DefinitionTree } from "../core/types";
+
+/**
+ * Generate dynamic bash completion script.
+ * Calls `wrangler __complete` at runtime for completions.
+ */
+export function getBashScript(_tree: DefinitionTree): string {
```

Comment:
Why is `_tree` passed to all these script functions but not used by any of them?